web/inheritance/extending_a_polymer_element/base_element.dart

```diff
@@ -1,5 +1,5 @@
 @HtmlImport('base_element.html')
```

Comment:
Lets just delete these as its no longer supported (even though it kind of works in some situations)